frontend/mr-admin-flate/src/components/avtaler/AvtaleSkjemaDetaljer.tsx

```diff
@@ -71,7 +67,7 @@ export function AvtaleSkjemaDetaljer({ tiltakstyper, ansatt, enheter, avtale }:
 
   const arenaOpphavOgIngenEierskap =
     avtale?.opphav === Opphav.ARENA &&
-    !migrerteTiltakstyperOgTiltakstyperUtenAvtaleIArena?.includes(watchedTiltakstype.arenaKode);
+    !migrerteTiltakstyper?.includes(watchedTiltakstype.arenaKode);
```

Comment:
Kallet tilt `migrerteTiltakstyper?.includes(watchedTiltakstype.arenaKode)` kan kanskje trekkes ut i en boolean som gjenbrukes alle sjekkene i stedet for å sjekke `includes` hvert sted?

Reply:
Ja, kan ordne det